public/apps/account/log-out-button.tsx

```diff
@@ -23,7 +23,11 @@ export function LogoutButton(props: { authType: string; http: HttpStart; divider
     return (
       <div>
         {props.divider}
-        <EuiButtonEmpty color="danger" size="xs" href="/auth/logout">
+        <EuiButtonEmpty
+          color="danger"
+          size="xs"
+          href={`${props.http.basePath.serverBasePath}/auth/logout`}
```

Comment:
Is this `serverBasePath` from config? If user inputs `/app/kibana/`, will that becomes `/app/kibana//auth/logout`?

Reply:
yes, the serverBasePath is the kibana base path configured in `kibana.yml`. And Kibana has validation to make sure  it doesn't end with slash

if user configure it to end with slash, Kibana will fail to start with following error
```
 FATAL  ValidationError: child "server" fails because [child "basePath" fails because ["basePath" with value "/_plugin/kibana/basepath/" fails to match the start with a slash, don't end with one pattern]]
```